src/modules/live-execution-tracker/live-execution-tracker.ts

```diff
@@ -109,6 +110,14 @@ export class LiveExecutionTracker {
     this._parentProcessInstanceId = await this._getParentProcessInstanceId();
     this._parentProcessModelId = await this._getParentProcessModelId();
 
+    const hasTaskId: boolean = routeParameters.taskId !== undefined
+      && routeParameters.taskId !== null && routeParameters.taskId !== '';
```

Comment:
```suggestion
      && routeParameters.taskId !== null
      && routeParameters.taskId !== '';
```

I would spend a line for each rule